src/components/promo/promo.less

```diff
@@ -0,0 +1,37 @@
+/**
+ * Promo Banner
+ */
+.promoMixin(@promoPrimaryColor) {
+  position: relative;
+  padding: 50px 0 50px 0;
+  color: @white;
+  background-color: @promoPrimaryColor;
+  text-align: center;
+
+  h3 {
+    font-size: 2.25rem;
+    margin-bottom: 1rem;
+    font-weight: 400;
+    margin: 20px 0 10px 0;
+  }
+
+  h4 {
+    font-size: 1.375rem;
+    line-height: 1.36;
+    font-weight: 400;
+  }
+
+  button {
+    background: @white;
+    border: 1px solid darken(@promoPrimaryColor, 15%);
+    border-radius: 40px;
+    font-family: 'Fira Sans', sans-serif;
+    font-weight: 500;
+    text-transform: uppercase;
+    color: @promoPrimaryColor;
+    width: auto;
+    height: auto;
+    margin: 1rem 0;
+    padding: 20px 50px;
+  }
```

Comment:
I found some [`button` rulesets in `splash.less`](https://github.com/mozilla/webmaker-desktop/blob/develop/src/pages/splash/splash.less#L27-L48) but I wasn't sure what those were for (there wasn't any `<button`> in `splash.jsx`)

Other than the rules provided in [guideline](https://cloud.githubusercontent.com/assets/1119821/8234436/ef93135a-15a9-11e5-9ad6-b1fa6c2beaad.png) I added 
```css
width: auto;
height: auto;
margin: 1rem 0;
padding: 20px 50px;
```
just to override what's already in `splash.less`